docs/aws.md

```diff
@@ -68,6 +68,7 @@ done
 
 # start Spark SQL client shell
 spark-sql --packages $DEPENDENCIES \
+    --conf spark.sql.defaultCatalog=my_catalog \
```

Comment:
In the same file there are multiple `spark-sql` commands, we need to add to all of these and also check in the other files? :)

Reply:
+1 to updating the other places with the same setting

Reply:
Agreed. This would be a good follow up.